app/screens/home/channel_list/categories_list/header/header.tsx

```diff
@@ -88,6 +88,13 @@ const getStyles = makeStyleSheetFromTheme((theme: Theme) => ({
         justifyContent: 'space-between',
         height: 40,
     },
+    outsideBox: {
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+    firstBox: {
+        width: '85%',
```

Comment:
Why 85%?

Reply:
I worked out this ratio value from the designs... it should have been 88% but 85% looks better.